src/runtime/runtime.cs

```diff
@@ -1991,6 +1996,9 @@ internal static IntPtr PyType_GenericAlloc(IntPtr type, long n)
         [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
         internal static extern void PyObject_GC_UnTrack(IntPtr tp);
 
+        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern void PyObject_ClearWeakRefs(IntPtr obj);
```

Comment:
We should try to use `*Reference` types for clarity for all new PInvoke functions.

Reply:
That makes sense, but can we just replace them all at once.😅

Reply:
@amos402 are you prepared to make this happen? )